mender/templates/deployments/_podtemplate.yaml

```
@@ -101,6 +102,33 @@ spec:
       value: "true"
     {{- end }}
     {{- end }}
+
+    {{- if .dot.Values.global.enterprise }}
+    {{- if and ( le (int (include "menderVersionMajor" .)) 3 ) ( lt (int (include "menderVersionMinor" .)) 7 ) }}
+    # redis_address deprecated from 3.7
+    - name: DEPLOYMENTS_REDIS_ADDR
+      value: {{ include "redis_address" . }}
+    - name: DEPLOYMENTS_REDIS_USERNAME
+      value: {{ .dot.Values.global.redis.username | default "" | quote }}
+    - name: DEPLOYMENTS_REDIS_PASSWORD
+      value: {{ .dot.Values.global.redis.password | default "" | quote }}
+    - name: DEPLOYMENTS_REDIS_DB
+      value: {{ .dot.Values.useradm.env.DEPLOYMENTS_REDIS_DB | default "1" | quote }}
+    - name: DEPLOYMENTS_REDIS_TIMEOUT_SEC
+      value: {{ .dot.Values.useradm.env.DEPLOYMENTS_REDIS_TIMEOUT_SEC | default "1" | quote }}
+    {{- else }}
+    {{- if not .dot.Values.global.redis.existingSecret }}
+    # redis_address deprecated from 3.7
+    - name: DEPLOYMENTS_REDIS_CONNECTION_STRING
+      value: {{ include "redis_connection_string" . }}
+    {{- end }}
+    - name: DEPLOYMENTS_REDIS_KEY_PREFIX
+      value: {{ .dot.Values.device_auth.env.DEPLOYMENTS_REDIS_KEY_PREFIX | default "deployments:v1" | quote }}
+    {{- end }}
+    - name: DEPLOYMENTS_REDIS_LIMITS_EXPIRE_SEC
+      value: {{ .dot.Values.device_auth.env.DEPLOYMENTS_REDIS_LIMITS_EXPIRE_SEC | default "3600" | quote }}
+    {{- end }}
```

There are only two `REDIS_` configuration variables for deployments
```suggestion
    - name: DEPLOYMENTS_REDIS_CONNECTION_STRING
      value: {{ include "redis_connection_string" . }}
    - name: DEPLOYMENTS_REDIS_KEY_PREFIX
      value: {{ .dot.Values.device_auth.env.DEPLOYMENTS_REDIS_KEY_PREFIX | default "deployments:v1" | quote }}
```